packages/legend-application-studio/src/stores/LegendStudioBaseStore.ts

```diff
@@ -188,6 +190,7 @@ export class LegendStudioBaseStore {
             this.applicationStore.navigationService.navigator.getCurrentAddress(),
           ),
         );
+        return false;
```

Comment:
can we add extended documentation here on why we do it like this, and why do we need to do this.

could be here, or could be near line 117. Thanks!

Reply:
I reworked this to make it cleaner